src/fn/upload-docs.js

```diff
@@ -34,7 +34,7 @@ const execute = async () => {
   }
 
   warn(`This operation will permanently write ${totalCount} docs.  Are you sure you want to continue?`);
-  if(!readline.keyInYN()) {
+  if(!environment.force && !readline.keyInYN()) {
```

Comment:
If we merge [pull/342](https://github.com/medic/medic-conf/pull/342) first, just a reminder to add the same check in these places:

- https://github.com/medic/medic-conf/pull/342/files#diff-8502033b92ecff583b9e274df1126b33R22
- https://github.com/medic/medic-conf/pull/342/files#diff-8502033b92ecff583b9e274df1126b33R13

Reply:
I almost wonder if it's worth the effort to wrap readline into our own module and perform the check within the module. Would alleviate having to add an additional check everywhere.

Reply:
I was thinking the same... it'd be way too easy to forget to check `environment.force` and break everyone's automation.

Reply:
I think it's a good idea to wrap readline!

Reply:
I have updated this to wrap readline into a module called userPrompt and updated everything. Can I get a re-review when someone is free?